src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -336,6 +336,26 @@ <h5>
                             ngbTooltip="{{ 'artemisApp.course.courseCommunicationSetting.messagingEnabled.tooltip' | artemisTranslate }}"
                         ></fa-icon>
                     </div>
+                    <div class="form-group" *ngIf="messagingEnabled">
+                        <label
+                            class="form-control-label"
+                            jhiTranslate="artemisApp.course.courseCommunicationSetting.messagingEnabled.codeOfConduct.label"
+                            for="field_messagingCodeOfConduct"
+                            >Messaging Code of Conduct
+                        </label>
+                        <fa-icon
+                            [icon]="faQuestionCircle"
+                            class="text-secondary"
+                            ngbTooltip="{{ 'artemisApp.course.courseCommunicationSetting.messagingEnabled.codeOfConduct.tooltip' | artemisTranslate }}"
+                        ></fa-icon>
+                        <textarea
+                            id="field_messagingCodeOfConduct"
+                            class="form-control"
+                            rows="2"
```

Comment:
Is this limited to two rows? We have other text areas that are resizable (e.g. when filing a complaint in an exercise), which would probably be useful here as well

Reply:
This is now a markdown editor